gradio/blocks.py

```diff
@@ -1656,10 +1669,18 @@ def validate_outputs(self, fn_index: int, predictions: Any | list[Any]):
 
     async def postprocess_data(
         self, fn_index: int, predictions: list | dict, state: SessionState | None
-    ):
+    ) -> Any:
```

Comment:
I see why you set it to Any to avoid type hinting entirely with the list | dict, but would be great to fix in a refactor soon